### ISSUE TITLE
Show teammate nametags

### ISSUE THREAD

**Issue by rubenwardy** [MT-CTF/capturetheflag#671]
This is possible using the HUD API

**Comment by Panquesito7** [comment]
Working on it.

**Comment by appgurueu** [comment]
Using waypoints I guess?

**Comment by HexyBubbled** [comment]
^^

**Comment by appgurueu** [comment]
@savilli what happened?

**Comment by farooqkz** [comment]
Maybe depends on https://github.com/minetest/minetest/issues/5996 ?

**Comment by magnetar47** [comment]
> Maybe depends on [minetest/minetest#5996](https://github.com/minetest/minetest/issues/5996) ?

That issue is about the regular player nametags, btw. We're using entity-based [`playertag`](https://github.com/MT-CTF/capturetheflag/tree/master/mods/other/playertag) instead.

**Comment by LoneWolfHT** [comment]
> > Maybe depends on [minetest/minetest#5996](https://github.com/minetest/minetest/issues/5996) ?
> 
> That issue is about the regular player nametags, btw. We're using entity-based [`playertag`](https://github.com/MT-CTF/capturetheflag/tree/master/mods/other/playertag) instead.

I would switch them back to the normal ones if I could control who they were shown to

**Comment by magnetar47** [comment]
> > > Maybe depends on [minetest/minetest#5996](https://github.com/minetest/minetest/issues/5996) ?
> > 
> > 
> > That issue is about the regular player nametags, btw. We're using entity-based [`playertag`](https://github.com/MT-CTF/capturetheflag/tree/master/mods/other/playertag) instead.
> 
> I would switch them back to the normal ones if I could control who they were shown to

I agree. It's simpler to do this, than to have entity-based playertags, and *then* somehow allow players to know where team-mates are in real-time.

**Comment by LoneWolfHT** [comment]
A globalstep revealing enemy names at a certain distance would be possible, otherwise the entity nametags could be kept.

**Comment by farooqkz** [comment]
It has been implemented
